app/services/profiles/update.rb

```diff
@@ -13,9 +13,6 @@ def initialize(profile, updated_attributes)
     end
 
     def call
-      # Ensure we have up to date attributes
```

Comment:
We should no longer need to do this. Profile fields are now automatically loaded on app boot and later updated by the `ProfileFields::Add` and `ProfileFields::Remove` service objects.